enterprise-suite/es-monitor-api/static-rules.yml

```diff
@@ -7,8 +7,8 @@
 - record: node_filesystem_free_percent
   expr: avg by (device, instance) ((node_filesystem_free{device=~"/.*"} / node_filesystem_size * 100) and (node_filesystem_size > 0))
 
-- record: node_network_errors
-  expr: sum by(instance) (node_network_transmit_errs + node_network_receive_errs + node_network_receive_drop + node_network_transmit_drop)
+- record: node_network_error_rate
```

Comment:
this name change must be accompanied by a change in the health rule that refers to the metric.  grep for "node_network_errors"

Reply:
you also need to update the smoke test, which looks for data with this metric name